catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -91,6 +101,39 @@ protected SolrCacheMBean getCacheProxy() {
         verify(mbean, times(1)).removeById(idsArray);
     }
 
+    @Test
+    public void testOverMaxItemList() throws Exception {
+        ConsoleOutput consoleOutput = mock(ConsoleOutput.class);
+        int totalResultsToDelete = 100;
+        int resultsPerCFQuery = 22;
+        int numCFCalls = 5;
+        List<String> ids = new ArrayList<>();
+        List<Metacard> mcList = getMetacardList(totalResultsToDelete);
+        for(int i = 0; i < mcList.size(); i++) {
```

Comment:
Might consider refactoring to foreach loop to make more concise and remove need for index.